assets/app/scripts/controllers/attachPersistentVolumeClaim.js

```diff
@@ -0,0 +1,280 @@
+                      return false;
+                    }
+                  }
+                }
+              }
+            }          
+          }
+          return true;
+        };
+
+        // breadcrumb must react depending on what we are attaching to (deployment or dc)
+        var rebuildBreadcrumb = function() {
+          $scope.breadcrumbs.splice(1, 0, {
+            title: "Deployments",
+            link: "project/" + $routeParams.project + "/browse/deployments"
+          });
+
+          var deploymentConfig = $scope.attach.deploymentConfig;
+          if (deploymentConfig) {
+            $scope.breadcrumbs.splice(2, 0, {
+              title: deploymentConfig.metadata.name,
+              link: "project/" + $routeParams.project + "/browse/deployments/" + deploymentConfig.metadata.name
+            });
+          }
+
+          var deployment = $scope.attach.deployment;
+          if (deployment) {
+            var deploymentVersion = $filter("annotation")(deployment, "deploymentVersion");
+            $scope.breadcrumbs.splice(2, 0, {
+              title: deploymentVersion ? "#" + deploymentVersion : deployment.metadata.name
+            });
+            var deploymentDeploymentConfigName = $filter("annotation")(deployment, "deploymentConfig");
+            if (deploymentDeploymentConfigName) {
+              $scope.breadcrumbs[2].link = "project/" + $routeParams.project + "/browse/deployments/" + deploymentDeploymentConfigName + "/" + deployment.metadata.name;
+              $scope.breadcrumbs.splice(2, 0, {
+                title: deploymentDeploymentConfigName,
+                link: "project/" + $routeParams.project + "/browse/deployments/" + deploymentDeploymentConfigName
+              });
+            }
+          }
+        };
+
+        var orderByDisplayName = $filter('orderByDisplayName');
+        var getErrorDetails = $filter('getErrorDetails');
+
+        if (!validate() || !load()) {
+          return;
+        }
+
+        $scope.attachPersistentVolumeClaim = function() {
+          $scope.disableInputs = true;
+
+          if ($scope.attachPersistentVolumeClaimForm.$valid) {
+            // generate a volume name if not provided
+            if (!$scope.attach.volumeName) {
+              $scope.attach.volumeName = StorageService.generateSimpleName("volume-");
+            }
+
+            var deploymentConfig = $scope.attach.deploymentConfig;
+            var deployment = $scope.attach.deployment;
+            var podTemplate = deploymentConfig ? deploymentConfig.spec.template : deployment.spec.template;
+
+            var persistentVolumeClaim = $scope.attach.persistentVolumeClaim;
+            var name = $scope.attach.volumeName;
+            var mountPath = $scope.attach.mountPath;
+
+            // we must attach to at least one container (han-picked by user, or just "all")
+            if (!containerToAttachProvided()) {
+              sendError("You must specify at least one container.");
```

Comment:
instead of sending this error to the top of the form, you should expand the containers list and have the error appear inline with that list